hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/addressbook/NodeCreateSuite.java

```diff
@@ -17,41 +17,344 @@
+            .build());
+    public static List<ServiceEndpoint> GOSSIP_ENDPOINTS_IPS =
+            Arrays.asList(endpointFor("192.168.1.200", 123), endpointFor("192.168.1.201", 123));
+    public static List<ServiceEndpoint> SERVICES_ENDPOINTS_IPS = Arrays.asList(endpointFor("192.168.1.205", 234));
+
+    /**
+     * This test is to check if the node creation fails during ingest when the admin key is invalid.
+     * @see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-869.md#specification">HIP-869</a>
+     */
+    @HapiTest
+    final Stream<DynamicTest> adminKeyIsInvalidOnIngest() {
+        return hapiTest(nodeCreate("nodeCreate")
+                .adminKeyName(NONSENSE_KEY)
+                .signedBy(GENESIS)
+                .hasPrecheck(KEY_REQUIRED)); // on ingest level before all the events on the handlers happens);
+        // expected status to reach consensus and this is the status */);
+    }
+
+    /**
+     * This test is to check if the node creation fails during pureCheck when the admin key is invalid.
+     * @see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-869.md#specification">HIP-869</a>
+     */
     @HapiTest
     @Tag(EMBEDDED)
-    final Stream<DynamicTest> createNodeWorks() {
-        final String description = "His vorpal blade went snicker-snack!";
+    final Stream<DynamicTest> adminKeyIsInvalidEmbedded() { // skipping ingest but purecheck still throw the same
+        return hapiTest(nodeCreate("nodeCreate")
+                .setNode("0.0.4") // exclude 0.0.3
+                .adminKeyName(NONSENSE_KEY)
+                .signedBy(GENESIS)
+                .hasKnownStatus(KEY_REQUIRED));
+    }
+
+    /**
+     * This test is to check if the node creation fails when the admin key is empty.
+     * @see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-869.md#specification">HIP-869</a>
+     */
+    @HapiTest
+    final Stream<DynamicTest> adminKeyIsEmpty() {
+        return hapiTest(nodeCreate("nodeCreate")
+                .adminKey(STANDIN_CONTRACT_ID_KEY)
+                .signedBy(GENESIS)
+                .hasPrecheck(KEY_REQUIRED));
+    }
+
+    /**
+     * This test is to check if the node creation fails when the admin key is not signed by the payer.
+     * @see <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-869.md#specification">HIP-869</a>
+     */
+    @HapiTest
+    @Tag(EMBEDDED)
+    final Stream<DynamicTest> adminKeyIsInvalidSigPayer() {
+        return hapiTest(
+                newKeyNamed("adminKey"),
+                cryptoCreate("payer").balance(ONE_HUNDRED_HBARS),
+                nodeCreate("nodeCreate")
+                        .setNode("0.0.4")
+                        .adminKeyName("adminKey")
+                        .signedBy("payer")
+                        .hasPrecheck(OK)
+                        .hasKnownStatus(INVALID_PAYER_SIGNATURE));
```

Comment:
What is this testing ?  The payer is `GENESIS` but signed by a different account ? I don't understand the comment
```
This test is to check if the node creation fails when the admin key is not signed by the payer.